sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -105,77 +106,50 @@ export class EventHubReceiver extends LinkEntity {
    * inside _onAmqpMessage.
-   * @protected
+   * @private
    */
-  protected _onMessage?: OnMessage;
+  private _onMessage?: OnMessage;
   /**
-   * @property _onError The error handler provided by the user that will be wrapped
+   * @property _OnError The error handler provided by the batching or streaming flavors of receive operations on the `EventHubConsumer`
    * inside _onAmqpError.
-   * @protected
+   * @private
    */
-  protected _onError?: OnError;
+  private _onError?: OnError;
   /**
-   * @property onAbort The aborter handler that will be invoked when user will call Aborter.abort()
-   * to cancel the receive request.
-   * @protected
+   * @property _checkpoint The sequence number of the most recently received AMQP message.
+   * @private
    */
-  protected _onAbort: () => void;
+  private _checkpoint: number = -1;
   /**
-   * @property _onAmqpError The message handler that will be set as the handler on the
-   * underlying rhea receiver for the "message" event.
-   * @protected
+   * @property _internalQueue A queue of events that were received from the AMQP link but not consumed externally.
+   * @private
    */
-  protected _onAmqpMessage: OnAmqpEvent;
+  private _internalQueue: ReceivedEventData[] = [];
   /**
-   * @property _onAmqpError The message handler that will be set as the handler on the
-   * underlying rhea receiver for the "receiver_error" event.
-   * @protected
+   * @property _usingInternalQueue Indicates that events in the internal queue are being processed.
+   * @private
    */
-  protected _onAmqpError: OnAmqpEvent;
+  private _usingInternalQueue: boolean = false;
   /**
-   * @property _onAmqpClose The message handler that will be set as the handler on the
-   * underlying rhea receiver for the "receiver_close" event.
-   * @protected
+   * @property _isReceivingMessages Indicates if messages are being received from this receiver.
    */
-  protected _onAmqpClose: OnAmqpEvent;
-  /**
-   * @property _onSessionError The message handler that will be set as the handler on
-   * the underlying rhea receiver's session for the "session_error" event.
-   * @protected
-   */
-  protected _onSessionError: OnAmqpEvent;
-  /**
-   * @property _onSessionClose The message handler that will be set as the handler on
-   * the underlying rhea receiver's session for the "session_close" event.
-   * @protected
-   */
-  protected _onSessionClose: OnAmqpEvent;
-  /**
-   * @property _checkpoint Describes sequenceNumber of the last message received.
-   * This is used as the sequenceNumber to receive messages from incase of recovery.
-   */
-  protected _checkpoint: number;
-  /**
-   * @property _aborter Describes Aborter instance that will be set by the user
-   * to cancel the request.
-   */
-  protected _abortSignal: AbortSignalLike | undefined;
+  private _isReceivingMessages: boolean = false;
 
   /**
    * @property Returns sequenceNumber of the last event received.
```

Comment:
```suggestion
   * @property Returns sequenceNumber of the last event received from the service. This will not match the last event received by `EventHubConsumer` when the `_internalQueue` is not empty
```